Casks/p/paste.rb

```diff
@@ -0,0 +1,19 @@
+cask "paste" do
+  version :latest
+  sha256 :no_check
+
+  url "https://hub.pasteapp.io/downloads/stable/latest"
+  name "Paste"
+  desc "Limitless clipboard"
+  homepage "https://pasteapp.io/"
+
```

Comment:
```suggestion

  livecheck do
    url "https://hub.pasteapp.io/downloads/stable/latest"
    regex(/Paste[._-]v?(\d+(?:\.\d+)+)\.zip/i)
    strategy :header_match do |headers, regex|
      match = headers["location"]&.match(regex)
      next if match.blank?

      match[1]
    end
  end

  auto_updates true
```

Reply:
It should be sufficient to use the default settings of the `HeaderMatch` strategy:

```suggestion

  livecheck do
    url "https://hub.pasteapp.io/downloads/stable/latest"
    strategy :header_match
  end

  auto_updates true
```